tests/Interface/core/ImageContainerTest.cpp

```diff
@@ -24,6 +24,38 @@
 #include <buddy/Core/Container.h>
 #include <buddy/DIP/ImageContainer.h>
 
+bool compare_flt(float a, float b) {
+  if (a == b)
+    return true;
+  float eps = std::abs(a - b);
+  return (eps < FLT_EPSILON);
+}
+
+template <typename T, size_t N>
+bool testImgcvnorm(cv::Mat testImgcv, Img<T, N> testImg, bool norm = false) {
+  int cvn = testImgcv.dims;
+  if (cvn != N)
+    return false;
+  for (size_t i = 0; i < N; i++) {
```

Comment:
Nit: Using pre-increment (++i instead of i++) is marginally better wherever you can (pls modify other places as well).